### ISSUE TITLE
export svg

### ISSUE THREAD

**Issue by linuxraudi** [kennycason/kumo#38]
Hi,

Is there a way to write to svg (or any other vector graphic)?

Andreas

**Comment by kennycason** [comment]
I currently don't have any plan to implement SVG output, but would be accepting of that added functionality. :)

**Comment by ibrahimrajabli** [comment]
still not svg export? :)

**Comment by kennycason** [comment]
Correct. It would require a full rebuild of the rendering issue as currently the rendering engine and collision is all based off of BufferedImage. I would need to think more about the implementation. Another idea is to simply store the location and size that the text is placed in the word cloud and from that i'm sure there's a library that can support text + SVG for the export. But I'd need to think how to do this cleanly. I've recently just been refactoring the core library to see if there is a better/cleaner/faster way to handle things.

**Comment by zachzientek** [comment]
+1 on being able to output SVG

**Comment by kennardconsulting** [comment]
There's a good chance all it would take is exposing those methods that create/accept a Graphics2D. Then we could hook in Apache Batik's Graphics2D implementation (which converts graphics calls to SVG).
